docs/code/local-setup.md

```diff
@@ -52,6 +53,8 @@ First, install Homebrew
 
 ```bash
 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+
+export PATH=/opt/homebrew/bin:/opt/homebrew/sbin:$PATH
```

Comment:
I don't think this does what you want, as it only temporarily updates the path, and may miss other things that homebrew wants to be initialized in your shell. The install script gives instructions for how to add the activation to your bash or zsh profile, did they not work for you?

Reply:
I think I missed those instructions personally.